cmd/experimental/kjobctl/apis/v1alpha1/volumebundle_types.go

```diff
@@ -22,7 +22,6 @@ import (
 )
 
 // VolumeBundleSpec defines the desired state of VolumeBundle
-// +kubebuilder:validation:XValidation:rule="self.mountPoints.map(x, x.name in self.volumes.map(y, y.name))", message="mountPoint name must match a volume name"
```

Comment:
@trasc @IrvingMg Looks like it's not need?

Reply:
My mistake, I think we still can use it but I forgot to update the property name. The rule should be 

```cel
// +kubebuilder:validation:XValidation:rule="self.containerVolumeMounts.map(x, x.name in self.volumes.map(y, y.name))", message="containerVolumeMount name must match a volume name"
```

Btw, notice that this rule works with an array of objects so this rule is also expensive to evaluate. If we can know the max of items, the better.

Reply:
No, doesn't work even if set MaxItems=1.

Reply:
I'm wondering, is it possible that this limitation only for test env?

Reply:
Because I can't see this error when installing crds to k8s.

Reply:
I think I found my mistake, I should use the `all` macro instead of `map`... Could you try with:

```
// +kubebuilder:validation:XValidation:rule="self.containerVolumeMounts.all(x, x.name in self.volumes.all(y, y.name))", message="containerVolumeMount name must match a volume name"
```

Reply:
Thanks